text/0063-doc-comments.md

```diff
@@ -0,0 +1,226 @@
+    action in [Action::"CreateList", Action::"GetLists"],
+    resource == Application::"TinyTodo"
+);
+```
+
+Doc comments may also be placed immediately prior schema elements to document the purpose of entity types and actions and before each attribute for an entity type or action context.
+
+```cedarschema
+/// This entity defines our central user type
+entity User { 
+    /// The location where this user works. E.g., ABC17 or XYZ77.
+    location : String,
+    /// A numeric level describing this users job.
+    jobLevel : Long,
+}
+```
+
+## Motivation
+
+Even though the syntax of Cedar policies is designed to be easy to read and understand, policies will still require some amount of documentation. 
+Cedar naturally supports comments so policy authors can leave remarks in their code, but these comments do not exist in our policy JSON format or in our internal AST.
+Automated generation of documentation from comments is a common capability found in almost all modern programming languages, so we should update Cedar to include automatically processable comments that support this use case.
+
+This RFC is largely motived by [this comment on RFC 48](https://github.com/cedar-policy/rfcs/pull/48#issuecomment-1937735924).
+It also addresses the request in [Cedar issue 660](https://github.com/cedar-policy/cedar/issues/660) for comments in the JSON policy format. 
+
+## Detailed design
+
+Doc comments are designed to be very similar to annotations, and could reasonably be implemented as syntactic sugar for a special annotation annotation key.
+The primary difference is that, in order to fully satisfy the documentation use case, doc comments need to appear in more positions than are supported for annotations.
+We may choose to extend annotation to be supported in all positions proposed for doc comments, but this is not a necessary extensions to accept this RFC.
+
+Doc comments will be defined by this grammar.
+
+```
+DocCommentLine ::= '///' [^\r\n]* [\r\n]
+DocComment ::= {DocCommentLine}
+```
+
+### Policies
+
+Documentation for the policy as whole appears before the policy effect.
+It may be anywhere in relation to the annotations on that policy.
+Documentation for individual policy scope clauses must appear immediately before that clause.
+
+The policy grammar is now:
+
+```
+Policy ::= (Annotation|DocComment)} Effect '(' Scope ')' {Conditions} ';'
+Scope ::= [DocComment] Principal ',' [DocComment] Action ',' [DocComment] Resource
+```
+
+### Schema
+
+Documentation for a declaration in a schema appears immediately before the `entity`, `action`, `type`, or `namespace` keyword at the start of the declaration.
+The fields of the attributes in a record type can also be documented with the comment appearing directly before the attribute name.
+A record type might be written for an entities attributes, actions context, or common type definition, or it might be nested inside of some other type.
```

Comment:
```suggestion
A record type might be written for an entity's attributes, action's context, or common type definition, or it might be nested inside of some other type.
```